features/nvstore/mbed_lib.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "nvstore",
+    "config": {
+        "enabled": {
+            "macro_name": "NVSTORE_ENABLED",
+            "value": true,
+            "help": "Enabled"
+        },
+        "max_keys": {
+            "macro_name": "NVSTORE_MAX_KEYS",
+            "value": 16,
+            "help": "Maximal number of allowed NVStore keys"
+        },
+        "area_1_address": {
+            "macro_name": "NVSTORE_AREA_1_ADDRESS",
+            "help": "Area 1 address"
+        },
+        "area_1_size": {
+            "macro_name": "NVSTORE_AREA_1_SIZE",
+            "help": "Area 1 size"
+        },
+        "area_2_address": {
+            "macro_name": "NVSTORE_AREA_2_ADDRESS",
+            "help": "Area 2 address"
+        },
+        "area_2_size": {
+            "macro_name": "NVSTORE_AREA_2_SIZE",
+            "help": "Area 2 size"
+        }
+    }
+}
```

Comment:
can we have some target defaults in here?

Reply:
By design, there are no target defaults in mbed_lin.json. By default, we take the last two sectors of the flash and use them for NVStore, one per area. If they are smaller than 4KB, we take a few consecutive ones to form areas >= 4KB. One can set values in mdeb_lib.json to override this default behavior, but that's entirely up to the specific application.